### ISSUE TITLE
Update to jquery 3.6.0 results in failure to render on browser

### ISSUE THREAD

**Issue by apeterson91** [rstudio/distill#421]
Hello,

I recently published a new article to my [blog](https://xstreetvalidated.com/) (shamless plug) and found 
that when I did the home page no longer rendered appropriately. Investigating further I found that the 
following line of code changed in my docs/index.html file:

```
<!-- original line of code -->
  <script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<!-- new line of code -->
  <script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
```

and that when changed back to the original jquery library, everything rendered appropriately. 

I would normally provide a reprex but I'm not sure how to do that with html files.  Please advise on how to best help isolate issues with this bug.

**Comment by cderv** [comment]
Hi ! 

Which version of **distill** are you using ? 

This issue due to a new **rmarkdown** version should have been fixed in last **distill** 1.3
https://github.com/rstudio/distill/blob/main/NEWS.md#distill-v13-cran

Can you check your version used ? 

Thanks.

**Comment by apeterson91** [comment]
That fixed it!  Was previously on 1.2, updating to 1.3 and building the site no longer results in the error.
